cypress/support/fragments/circulation/newNoticePolicy.js

```diff
@@ -0,0 +1,112 @@
+import getRandomPostfix from '../../utils/stringTools';
+import {
+  Button,
+  TextArea,
+  NavListItem,
+  Checkbox,
+  Select,
+  Section,
+  Link,
+  TextInput,
+  Heading
+} from '../../../../interactors';
+
+const actionsButton = Button('Actions');
+const addNoticeButton = Button('Add notice');
+const nameField = TextInput({ id: 'notice_policy_name' });
+const descriptionField = TextArea({ id: 'notice_policy_description' });
+const sections = {
+  section1: Section({ id: 'editLoanNotices' }),
+  section2: Section({ id: 'editRequestNotices' }),
+  section3: Section({ id: 'editFeeFineNotices' }),
+};
+
+export default {
+  defaultUi: {
+    name: `Test_notice_${getRandomPostfix()}`,
+    description: 'Created by autotest team',
+  },
+
+  create(patronNoticePolicy) {
+    this.checkForm();
+    this.fillGeneralInformation(patronNoticePolicy);
+  },
+
+  startAdding() {
+    cy.do(Button({ id: 'clickable-create-entry' }).click());
+  },
+  checkForm() {
+    cy.expect([
+      Heading('New patron notice policy').exists(),
+      nameField.exists(),
+      nameField.has({ value: '' }),
+      descriptionField.exists(),
+      descriptionField.has({ value: '' }),
+      Checkbox({ id: 'notice_policy_active' }).has({ checked:false }),
+    ]);
+    Object.values(sections).forEach((specialSection) => cy.expect(specialSection.find(addNoticeButton).has({ disabled: false, visible: true })));
+  },
+  addNotice(patronNoticePolicy) {
```

Comment:
can't see steps related with

- Email format is selected
- Select "Check out"

Reply:
lines 53 - 54